edgecast/internal/ecclient/ec_client.go

```diff
@@ -61,6 +70,21 @@ func (c ECClient) SubmitRequest(params SubmitRequestParams) (*Response, error) {
 	return resp, nil
 }
 
+func scrubSensitiveHeaders(m map[string]string) map[string]string {
+	scrubbed := make(map[string]string, len(m))
```

Comment:
Hey Steven! Please take a look at this playground. I don't believe we need a new map. 
https://go.dev/play/p/G8Gxm78px-p

Reply:
@wdawg33 I made a deep copy so that the original map is unmodified. Check this, notice the extra print statement: https://go.dev/play/p/1uVLwGrq-YE

Reply:
Understood, we are printing before using the map.